receiver/opencensusreceiver/octrace/opencensus.go

```diff
@@ -95,7 +69,7 @@ const receiverTagValue = "oc_trace"
 // OpenCensus-traceproto compatible libraries/applications.
 func (ocr *Receiver) Export(tes agenttracepb.TraceService_ExportServer) error {
 	// We need to ensure that it propagates the receiver name as a tag
-	ctxWithReceiverName := observability.ContextWithReceiverName(tes.Context(), receiverTagValue)
```

Comment:
nit: is the constant still used anywhere? if not delete it. Related ideally this should be the name of the receiver in case there are more than one configured (atypical but possible)

Reply:
Fixing.